mod-source-record-manager-server/src/test/java/org/folio/services/JobExecutionProgressServiceImplTest.java

```diff
@@ -170,7 +168,6 @@ public void shouldUpdateProgress(TestContext context) {
         context.assertFalse(jobMonitoring.getNotificationSent());
         async.complete();
       });
-      async.complete();
     });
   }
 
```

Comment:
Class JobExecutionProgressMutator can be deleted since it has not been used

Reply:
Thank you @SerhiiNosko , deleted it.